tests/Aspire.Hosting.Tests/Oracle/OracleDatabaseFunctionalTests.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Tests.Helpers;
+using Xunit;
+
+namespace Aspire.Hosting.Tests.Oracle;
+
+[Collection("IntegrationServices")]
+public class OracleDatabaseFunctionalTests
+{
+    private readonly IntegrationServicesFixture _integrationServicesFixture;
+
+    public OracleDatabaseFunctionalTests(IntegrationServicesFixture integrationServicesFixture)
+    {
+        _integrationServicesFixture = integrationServicesFixture;
+    }
+
+    [LocalOnlyFact()]
+    public async Task VerifyOracleDatabaseWorks()
+    {
+        var testProgram = _integrationServicesFixture.TestProgram;
+        var client = _integrationServicesFixture.HttpClient;
+
+        using var cts = new CancellationTokenSource(TimeSpan.FromMinutes(1));
+
+        var response = await testProgram.IntegrationServiceABuilder!.HttpGetAsync(client, "http", "/oracledatabase/verify", cts.Token);
+
+        Assert.True(response.IsSuccessStatusCode);
```

Comment:
```suggestion
        var response = await testProgram.IntegrationServiceABuilder!.HttpGetAsync(client, "http", "/oracledatabase/verify", cts.Token);
        var responseContent = await response.Content.ReadAsStringAsync();

        Assert.True(response.IsSuccessStatusCode, responseContent);
```

This way if the test fails, the log will get what went wrong.

Reply:
Nice